### PR TITLE
News submission page

### PR DESCRIPTION
**Pull request by faithakinbobola**
Added news submission page

### CODE REVIEW

File: submission.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Virametric</title>
```
Comment (goke-epapa) [inline submission.html]: I'll suggest you add more contact on the name of the page
```suggestion
        <title>Virametric - News Submission</title>
```

File: thankyou.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Submission</title>
```
Comment (goke-epapa) [inline thankyou.html]:
```suggestion
        <title>Thank you for your Submission</title>
```

Comment (goke-epapa) [inline thankyou.html]:
```suggestion
        <title> Virametric  - Thanks for your Submission</title>
```

File: thankyou.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    </head>
+    <body>
+        <h1>Thank you for keeping us updated. We hope to hear from you some other time.</h1>
+    </body>
+</html>
```
Comment (goke-epapa) [inline thankyou.html:19]: Add a new line after this closing tag

**Review by goke-epapa — APPROVED** [review]

File: thankyou.html (hunk 3 above)
Comment (faithakinbobola) [reply thankyou.html:19]: i don't understand what you mean by this.

Comment (goke-epapa) [reply thankyou.html:19]: After the last character i.e. `>`, just press the enter button.
Feel free to merge if you don't get it now, we can go through it during the call later.

**Review by JoyChukwukere — APPROVED** [review]

**Pull request merged**